hasura/migrations/default/1665513818085_alter_table_public_users_add_column_entrance/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."users" add column "entrance" text
```

Comment:
https://www.postgresql.org/docs/14/datatype-character.html

We don't need this to be a `text` type, just a `varchar` should be good, these will all be short strings.

Reply:
done

Reply:
@sshmm we can remove this entire migraiton file now 1665513818085, since you added a new migraiton with the new type.

Reply:
I removed the other one and updated the types in the first one